Gemfile

```diff
@@ -10,8 +10,8 @@ group :test do
   gem 'hashdiff'
   gem 'minitest'
   gem 'mirage'
-  gem 'rake'
-  gem 'pronto'
+  gem "rake"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.